edx-platform/pearson-vue-theme/lms/templates/header/navbar-not-authenticated.html

```diff
@@ -9,29 +9,38 @@
 from django.urls import reverse
 from django.utils.translation import ugettext as _
 from six import text_type
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 
 <%
+  theme = configuration_helpers.get_value('theme', '')
```

Comment:
I think this variable is no longer used on this template. @luismorenolopera

Reply:
removed